### PR TITLE
Add notebook links to schedule

### PR DESCRIPTION
**Pull request by sjspielman**
Towards #9 

I've added completed notebook and exercise links to the schedule. I wasn't 100% sure about how to include the nb RNAseq notebooks; in particular note the **TODO** I have in there to catch during review. Let me know where I need to move links around!

### CODE REVIEW

File: workshop/SCHEDULE.md [hunk 1]
```diff
@@ -56,32 +56,34 @@ Minneapolis, MN 55414 in Room 105.
...
-| 9:00 AM     | RNA-Seq quantification (trim, filter, quant) |
-| 10:00 AM    | RNA-Seq import with tximeta |
+| 9:00 AM     | [RNA-Seq quantification (trim, filter, quant)](../completed-notebooks/RNA-seq/01-qc_trim_quant.nb.html) |
+| 10:00 AM    | [RNA-Seq import with tximeta](../completed-notebooks/RNA-seq/02-gastric_cancer_tximeta.nb.html) |
 | 10:45 AM    | Coffee break |
-| 11:00 AM    | RNA-Seq exploratory analysis |
-|             | Exercise: Neuroblastoma cell line tximeta |
-|             | Exercise: Exploratory data analysis |
+| 11:00 AM    | [RNA-Seq exploratory analysis](../completed-notebooks/RNA-seq/03-gastric_cancer_exploratory.nb.html) |
+|             | **TODO this isn't an exercise anymore?** [Exercise: Neuroblastoma cell line tximeta](../completed-notebooks/RNA-seq/04-nb_cell_line_tximeta.nb.html) |
```
Comment (jashapiro) [inline workshop/SCHEDULE.md:63]: This is an "in class exercise." The link you have here is to the instructions which is fine, but the "completed" notebook is in the `setup` directory and should probably go into magic files.

File: workshop/SCHEDULE.md [hunk 2]
```diff
@@ -56,32 +56,34 @@ Minneapolis, MN 55414 in Room 105.
...
 | **Day 4**   | **2024-08-22 <br> Bulk RNA-Seq: Pathway Analysis <br> Participant Presentations** |
 | 8:30 AM     | Breakfast and coffee available |
 | 9:00 AM     | Questions and review |
-| 9:30 AM     | Introduction to pathway analysis <br> [Introduction to Pathway Analysis slides (PDF)](../slides/2024-08-22_intro-to-pathway-analysis.pdf) |
-| 10:00 AM    | Pathway analysis: Over-representation analysis |
+| 9:30 AM     | Introduction to pathway analysis <br> [Introduction to pathway analysis slides (PDF)](../slides/2024-08-22_intro-to-pathway-analysis.pdf) |
+| 10:00 AM    | [Pathway analysis: Over-representation analysis](../completed-notebooks/pathway-analysis/01-overrepresentation_analysis.nb.html) |
 | 11:00 AM    | Coffee break |
-| 11:15 AM    | Pathway analysis: <br> Gene Set Enrichment Analysis <br> Gene Set Variation Analysis |
+| 11:15 AM    | Pathway analysis: <br> [Gene set enrichment analysis](../completed-notebooks/pathway-analysis/02-gene_set_enrichment_analysis.nb.html) <br> [Gene Set Variation Analysis](../completed-notebooks/pathway-analysis/03-gene_set_variation_analysis.nb.html) |
```
Comment (jashapiro) [inline workshop/SCHEDULE.md:82]: This was capitalized for acronym/method name reasons, but I am kind of agnositc. I'd probably take off the `Pathway analysis:` prefix on these though (and ORA). 


```suggestion
| 11:15 AM    |  [Gene Set Enrichment Analysis](../completed-notebooks/pathway-analysis/02-gene_set_enrichment_analysis.nb.html) <br> [Gene Set Variation Analysis](../completed-notebooks/pathway-analysis/03-gene_set_variation_analysis.nb.html) |
```

File: workshop/SCHEDULE.md (hunk 1 above)
Comment (jaclyn-taroni) [reply workshop/SCHEDULE.md:63]: It should already be in magic files, but let me know if that's not true (https://github.com/AlexsLemonade/training-admin/issues/1132#issuecomment-2274039079).

Comment (jaclyn-taroni) [reply workshop/SCHEDULE.md:63]: Ah, you don't mean the output necessarily - I see.

Comment (sjspielman) [reply workshop/SCHEDULE.md:63]: > the "completed" notebook is in the setup directory and should probably go into magic files.

I've copied our `RNA-seq/setup/ref_notebooks/04*` notebooks over, (for now?) living in `/shared/data/training-data/2024-august/RNA-seq/ref-notebooks_nb_cell_line`. Eh?

Comment (sjspielman) [reply workshop/SCHEDULE.md]: I guess we probably only need the `Rmd` copied over but wasn't 100% sure, so did both for now.

File: workshop/SCHEDULE.md [hunk 3]
```diff
@@ -56,32 +56,34 @@ Minneapolis, MN 55414 in Room 105.
...
-| 9:00 AM     | RNA-Seq quantification (trim, filter, quant) |
-| 10:00 AM    | RNA-Seq import with tximeta |
+| 9:00 AM     | [RNA-Seq quantification (trim, filter, quant)](../completed-notebooks/RNA-seq/01-qc_trim_quant.nb.html) |
+| 10:00 AM    | [RNA-Seq import with tximeta](../completed-notebooks/RNA-seq/02-gastric_cancer_tximeta.nb.html) |
 | 10:45 AM    | Coffee break |
-| 11:00 AM    | RNA-Seq exploratory analysis |
-|             | Exercise: Neuroblastoma cell line tximeta |
-|             | Exercise: Exploratory data analysis |
+| 11:00 AM    | [RNA-Seq exploratory analysis](../completed-notebooks/RNA-seq/03-gastric_cancer_exploratory.nb.html) |
+|             | [Exercise: Neuroblastoma cell line tximeta](../completed-notebooks/RNA-seq/04-nb_cell_line_tximeta.nb.html) |
```
Comment (jashapiro) [inline workshop/SCHEDULE.md:63]: Maybe we rename this for distinction?
```suggestion
|             | [Activity: Neuroblastoma cell line tximeta](../completed-notebooks/RNA-seq/04-nb_cell_line_tximeta.nb.html) |
```

File: workshop/SCHEDULE.md [hunk 4]
```diff
@@ -56,32 +56,34 @@ Minneapolis, MN 55414 in Room 105.
...
 | **Day 4**   | **2024-08-22 <br> Bulk RNA-Seq: Pathway Analysis <br> Participant Presentations** |
 | 8:30 AM     | Breakfast and coffee available |
 | 9:00 AM     | Questions and review |
-| 9:30 AM     | Introduction to pathway analysis <br> [Introduction to Pathway Analysis slides (PDF)](../slides/2024-08-22_intro-to-pathway-analysis.pdf) |
-| 10:00 AM    | Pathway analysis: Over-representation analysis |
+| 9:30 AM     | Introduction to pathway analysis <br> [Introduction to pathway analysis slides (PDF)](../slides/2024-08-22_intro-to-pathway-analysis.pdf) |
+| 10:00 AM    | [Over-representation analysis](../completed-notebooks/pathway-analysis/01-overrepresentation_analysis.nb.html) |
 | 11:00 AM    | Coffee break |
-| 11:15 AM    | Pathway analysis: <br> Gene Set Enrichment Analysis <br> Gene Set Variation Analysis |
+| 11:15 AM    |  [Gene Set Enrichment Analysis (GSEA)](../completed-notebooks/pathway-analysis/02-gene_set_enrichment_analysis.nb.html) <br> [Gene Set Variation Analysis (GSVA)](../completed-notebooks/pathway-analysis/03-gene_set_variation_analysis.nb.html) |
```
Comment (jashapiro) [inline workshop/SCHEDULE.md:82]:
```suggestion
| 11:15 AM    | [Gene Set Enrichment Analysis (GSEA)](../completed-notebooks/pathway-analysis/02-gene_set_enrichment_analysis.nb.html) <br> [Gene Set Variation Analysis (GSVA)](../completed-notebooks/pathway-analysis/03-gene_set_variation_analysis.nb.html) |
```

**Review by jashapiro — APPROVED** [review]
This looks fine, with one wording change suggestion.  I believe this is all independent of the changes in file location mentioned in https://github.com/AlexsLemonade/training-modules/pull/797

File: workshop/SCHEDULE.md (hunk 3 above)
Comment (sjspielman) [reply workshop/SCHEDULE.md:63]: good idea

**Pull request merged**